code/Model/Submission/SubmittedFileField.php

```diff
@@ -41,14 +41,24 @@ public function getFormattedValue()
         $name = $this->getFileName();
         $link = $this->getLink();
```

Comment:
```suggestion
        $link = $this->getLink(false);
```

Getting the Link automatically session grant you the file, even if it's not outputted on the page.